module/metrics/hotstuff.go

```diff
@@ -166,24 +196,36 @@ func (hc *HotstuffCollector) HotStuffWaitDuration(duration time.Duration, event
 	hc.waitDuration.WithLabelValues(event).Observe(duration.Seconds()) // unit: seconds; with float64 precision
 }
 
-// HotstuffCollector reports Metrics C8: Current View
+// CountSkipped counts the number of skips we did.
+func (hc *HotstuffCollector) CountSkipped() {
```

Comment:
yeah, we could do it like this, `CountSkipped` here tracks how many times we have used QC to change view instead of TC. i think it is still a useful metric to have. Maybe let's just rename `CountSkipped` to more obvious name?